erniebot-agent/src/erniebot_agent/retrieval/baizhong_search.py

```diff
@@ -42,7 +42,14 @@ def __init__(
 
         """
         self.base_url = os.getenv("AISTUDIO_BASE_URL", self._AISTUDIO_BASE_URL)
-        self.access_token = access_token
+        global_access_token = os.getenv("EB_AGENT_ACCESS_TOKEN")
+        if global_access_token is not None:
+            self.access_token = global_access_token
+        elif access_token is not None:
+            self.access_token = access_token
+        else:
+            raise BaizhongError("You must provide either a access_token or EB_AGENT_ACCESS_TOKEN.")
```

Comment:
Please ensure that either an access_token is provided or the EB_AGENT_ACCESS_TOKEN is set up as an environment variable.